blank-canvas-blocks/experimental-theme.json

```diff
@@ -168,6 +168,14 @@
 						"lineHeight": 1.125,
 						"fontFamily": "var(--wp--preset--font-family--headings)"
 					}
+				},
+				"verse": {
+					"padding": {
```

Comment:
Could this just be one variable instead of one each for top right bottom left?

Reply:
I like the idea of those being separate. I used WordPress/gutenberg#27579 as a reference.